src/sql-parser/src/ast/defs/ddl.rs

```diff
@@ -456,7 +456,16 @@ impl_display!(DbzMode);
 
 #[derive(Debug, Clone, PartialEq, Eq, Hash, EnumKind)]
 #[enum_kind(ConnectorType)]
-pub enum CreateSourceConnector {
+pub enum CreateConnector<T: AstInfo> {
+    KafkaBroker {
+        broker: String,
+        with_options: Vec<SqlOption<T>>,
+    },
+}
+
+#[derive(Debug, Clone, PartialEq, Eq, Hash, EnumKind)]
+#[enum_kind(SourceInstanceType)]
+pub enum CreateSourceInstance {
```

Comment:
What does "instance" here signify?

Reply:
Same idea as above, this is all of the pieces of an actual source that has its own distinct state.